src/Terminal.jsx

```diff
@@ -1406,25 +1410,34 @@ let Terminal = {
                     if (commandArray.length < 2) {
                         postError("Incorrect number of arguments. Usage: tail [script] [arg1] [arg2]...");
                     } else {
-                        const scriptName = Terminal.getFilepath(commandArray[1]);
-                        if (!isScriptFilename(scriptName)) {
-                            postError("tail can only be called on .script files (filename must end with .script)");
-                            return;
-                        }
+                        if(typeof commandArray[1] === 'string') {
```

Comment:
else-if chain can be merged (body of else consists of nothing but if statements

Reply:
Alternatively, put a return and remove the else since all other error messages seem to follow that format. Easier to follow that way.